android/src/main/java/com/reactnativecommunity/webview/RNCWebViewModule.java

```diff
@@ -32,19 +35,29 @@
 import java.io.File;
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.List;
 
 import static android.app.Activity.RESULT_OK;
 
 @ReactModule(name = RNCWebViewModule.MODULE_NAME)
 public class RNCWebViewModule extends ReactContextBaseJavaModule implements ActivityEventListener {
   public static final String MODULE_NAME = "RNCWebView";
-  private static final int PICKER = 1;
   private static final int PICKER_LEGACY = 3;
+  private static final int OPEN_CAMERA = 4;
+  private static final int SELECT_FILE = 5;
   private static final int FILE_DOWNLOAD_PERMISSION_REQUEST = 1;
+  private static final int CAMERA_PERMISSION_REQUEST = 2;
+
   final String DEFAULT_MIME_TYPES = "*/*";
+  final String TAKE_PHOTO = "Take a photo…";
+  final String TAKE_VIDEO = "Record a video…";
+  final String CHOOSE_FILE = "Choose an existing file…";
+  final String CANCEL = "Cancel";
+
```

Comment:
We need support multi language, so please put the texts to `strings.xml` file

Reply:
I have made the changes. I am not sure about loading resources correctly in the given context. Could you please have a loot at it?